packages/astro/src/core/errors/errors-data.ts

```diff
@@ -17,6 +17,7 @@ export const AstroErrorData = defineErrors({
 	UnknownCompilerError: {
 		title: 'Unknown compiler error.',
 		code: 1000,
+		hint: 'This is almost always a problem with the Astro compiler, not your code. Please open an issue at https://astro.build/issues.',
```

Comment:
Do we want to add a new vanity link like https://astro.build/issues/compiler so it directs to the compiler repo instead?

Reply:
https://github.com/withastro/astro.build/pull/506